spec/requests/feeds_spec.rb

```diff
@@ -32,6 +32,13 @@
         assert_valid_feed
       end
     end
+
+    it "updates last_fetched" do
+      feed.update(last_fetched: nil)
+      get feed_url(feed, format: :atom)
+      feed.reload
+      expect(feed.last_fetched).not_to be_nil
```

Comment:
I am not sure if there's a better way to test this but maybe it would be nice to validate the value is in fact a date?